test/utils/currency.js

```diff
@@ -148,4 +150,89 @@ describe('the currency utility module', () => {
         .equal('฿523.12000000');
     });
   });
+
+  describe('has functions that involve converting between currencies', () => {
+    let ajax;
+
+    before(function () {
+      ajax = sinon.stub($, 'ajax', () => {
```

Comment:
Yeah, cool stuff here... using Sinon to stub ajax, so when I call fetchExchangeRates(), it doesn't blow up without a server, it calls the stubbed function which I have set to return some dummy data.